### PR TITLE
Removed npm test error

### LINKED ISSUE #1999: Travis Ci build and npm test failing
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Description :**
Travis Ci build and npm test failing due to addition of a duplicate key in create-badge.js

**Current behaviour:**
- npm test shows 1 fail
- travis ci showing build failed 

**Expected behaviour:**
Should not fail the test

**Steps to reproduce:**
 - Update you branch
 - Run npm test in badgeyay/frontend and see the error

**Screenshots**
screenshot from 2019-02-06 21-36-31

screenshot from 2019-02-07 00-30-28

**Would you like to work on the issue?**
 Yes

### PR DESCRIPTION
**Pull request by creyente1897**
Fixes #1999 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.

#### Changes proposed in this pull request:
- Removed npm test error due to which travis ci build was failing

### Screenshots for the change

screenshot from 2019-02-07 03-06-40

### CODE REVIEW

File: frontend/app/controllers/create-badges.js [hunk 1]
```diff
@@ -616,9 +616,6 @@ export default Controller.extend({
 
     },
 
-    defaultlogoimage() {
-    },
```
Comment (abishekvashok) [inline frontend/app/controllers/create-badges.js]: Maybe comment it out as that is a `should make one test for`

Comment (creyente1897) [reply frontend/app/controllers/create-badges.js]: Sir the error was of a duplicate key as the defaultlogoimage() function already exists in create-badge.js so should i change the function name to defaultLogoImage()?

**Comment by abishekvashok** [comment]
I am sorry, remove it then

**Comment by abishekvashok** [comment]
Thanks. Squashed and merged :P

**Comment by creyente1897** [comment]
@abishekvashok  Thank you sir :)

**Pull request merged**